velox/common/memory/MemoryPool.h

```diff
@@ -890,6 +896,68 @@ class MemoryPoolImpl : public MemoryPool {
   // The number of internal memory reservation collisions caused by concurrent
   // memory reservation requests.
   std::atomic<uint64_t> numCollisions_{0};
+
+#define DEBUG_RECORD_ALLOC(...)  \
+  if (debugMode_) {              \
```

Comment:
```
if (FOLLY_UNLIKELY(debugMode_)) {
```

Reply:
I didn't put this here because adding unlikely will likely make the opposite condition evaluation slower, ending in slower debug mode. It is a direct bool eval I think compared with other parts of the allocation/free code it is negligible. What do you think?